packages/blocks/src/root-block/widgets/slash-menu/styles.ts

```diff
@@ -5,6 +5,7 @@ import { scrollbarStyle } from '../../../_common/components/utils.js';
 
 export const styles = css`
   .overlay-mask {
+    pointer-events: auto;
```

Comment:
because the peek view container has `pointer-events: none;` ?

Reply:
Yes. I think the issue is caused by this line:
https://github.com/radix-ui/primitives/blob/main/packages/react/dismissable-layer/src/DismissableLayer.tsx#L117

When radixui dialog opens, it will applies pointer-events: none to body. All clickable elements should be explicitly marked as pointer-events: auto